examples/readiness-probe-pod

```diff
@@ -0,0 +1,19 @@
+# Copy of pod.yaml without file extension for test
```

Comment:
I slightly prefer keeping the example simple, but I'm not really against it. Just make sure you expalin what it is, if there's an accompanying example doc.